client_cli/src/main.rs

```diff
@@ -420,6 +422,36 @@ mod domain {
             Ok(())
         }
     }
+
+    /// Transfer a domain between accounts
+    #[derive(Debug, StructOpt)]
+    pub struct Transfer {
+        /// Domain name as double-quited string
+        #[structopt(short, long)]
+        pub id: DomainId,
+        /// Account from which to transfer (in form `name@domain_name')
+        #[structopt(short, long)]
+        pub from: AccountId,
+        /// Account from which to transfer (in form `name@domain_name')
+        #[structopt(short, long)]
+        pub to: AccountId,
+        /// The JSON/JSON5 file with key-value metadata pairs
+        #[structopt(short, long, default_value = "")]
+        pub metadata: super::Metadata,
```

Comment:
What is this metadata for?

Reply:
Yes, seems this argument is not needed. By the way, I've found out that there are other subcommands which accepts metadata that I don't think is needed there. For instance, `asset transfer` or `peer unregister` shouldn't accept metadata in my opinion. Let's fix them in a separate pull request?

Reply:
AFAIU this metadata is used in transaction. So why to remove it?